pkg/identity/identity.go

```diff
@@ -58,7 +58,7 @@ type FullIdentity struct {
 type SetupConfig struct {
 	CertPath  string `help:"path to the certificate chain for this identity" default:"$IDENTITYDIR/identity.cert"`
 	KeyPath   string `help:"path to the private key for this identity" default:"$IDENTITYDIR/identity.key"`
-	Overwrite bool   `help:"if true, existing identity certs AND keys will overwritten for" default:"false"`
+	Overwrite bool   `help:"if true, existing identity certs AND keys will overwritten for" default:"false" setup:"true"`
```

Comment:
Do we still want to support overwrite here?

Reply:
I think so but I don't have a strong opinion about it. I feel like it's best to make as minimal changes to these configs as possible until we have tests for the compiled binaries.

Reply:
We removed overwrite from other tools. I think it's probably safer not to have them. Rather than have overwrite I would display a message: `X already exists, keeping old for safety. If you want to erase your X, please manually delete files "Y", "Z", "W"`.